src/components/Disclaimer/index.js

```diff
@@ -62,10 +55,15 @@ const DisclaimerModal = () => {
             decisions.
           </p>
           <div className="mb-3 highlight-txt">
-            <input type="checkbox" className="number" value="One" onChange={handleCheckedFirst} />
+            <input type="checkbox" className="number" value="One" onChange={(e) => setCheckedFirst(e.target.checked)} />
```

Comment:
thats not what i meant.. i said to move it into the hook.. not into the jsx/html..

move those two functions into the useEffect hook

Reply:
When I write the two functions like this, It works, we can the functions below the return statement.

` const handleFirstCheck = (e) => {
    setCheckedFirst(e.target.checked)
  }
  const handleSecondCheck = (e) => {
    setCheckedSecond(e.target.checked)
  }
  useEffect(() => {
    if (isCheckedFirst && isCheckedSecond) {
      setBtnDisable(false)
    }
  }, [isCheckedFirst, isCheckedSecond])`

But when we write functions like this, it isn't working, because we can't use the functions outside the return statement.

`  useEffect(() => {
    if (isCheckedFirst && isCheckedSecond) {
      setBtnDisable(false)
    }
    const handleFirstCheck = (e) => {
      setCheckedFirst(e.target.checked)
    }
    const handleSecondCheck = (e) => {
      setCheckedSecond(e.target.checked)
    }
  }, [isCheckedFirst, isCheckedSecond])`

That's why I wrote the functions like the first code :)